# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/uclibc/hotfix-mfdpic.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Assigning CPU_CFLAGS in Rules.mak overwrites the already pre-fill with
-fno-builtin and friends and thus fails to link ldso on modern gccc with
undefined references to memcpy et al.

--- uClibc-ng-1.0.39/Rules.mak.vanilla	2021-10-07 13:11:29.593860466 +0200
+++ uClibc-ng-1.0.39/Rules.mak	2021-10-07 13:12:00.672862682 +0200
@@ -492,7 +492,7 @@
 ifeq ($(TARGET_ARCH),bfin)
 	SYMBOL_PREFIX=_
 ifeq ($(UCLIBC_FORMAT_FDPIC_ELF),y)
-	CPU_CFLAGS-y:=-mfdpic
+	CPU_CFLAGS-y+=-mfdpic
 	CPU_LDFLAGS-y += -Wl,-melf32bfinfd
 	PICFLAG:=-fpic
 	PIEFLAG_NAME:=-fpie
